src/jit/compiler.h

```diff
@@ -2274,7 +2274,11 @@ public :
 #endif // !LEGACY_BACKEND
 
     void                lvaAssignVirtualFrameOffsetsToArgs();
-    int                 lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize, int argOffs);
+    int                 lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize, int argOffs
+#ifdef UNIX_AMD64_ABI
```

Comment:
This is a better approach than using:
#ifdef
#else
#endif

Similar to DEBUGARG